life_expectancy/cleaning.py

```diff
@@ -0,0 +1,101 @@
+""" The code in this file loads, cleans and saves a dataset """
+from pathlib import Path
+import argparse
+import pandas as pd
+
+#--------------------------------------------------------------------
+# VARIABLES
+#--------------------------------------------------------------------
+
+CURRENT_DIR = str(Path(__file__).parent)
+PATH_RAW_DATASET = f"{CURRENT_DIR}/data/eu_life_expectancy_raw.tsv"
+PATH_CLEAN_DATASET = f"{CURRENT_DIR}/data/pt_life_expectancy.csv"
```

Comment:
Emanuel did the same comment to me. I'm going to quote I'm actually 🙂 "If you filter for a region that is not 'PT', the output will still be saved as 'pt_life_expectancy.csv". I would suggest making the name of the outputted *.csv file dynamic (in accordance with the selected region)."